docs/reference/manifest/group.md

```diff
@@ -35,7 +35,7 @@ Required. The label of the group. The **resid** attribute can be no more than 32
 
```

Comment:
In the "Child elements" table above, can you note that this doesn't apply to Outlook? Since it says "Icon" is required, readers may assume without going further that it's required for Outlook too.